cross/libaom/Makefile

```diff
@@ -52,3 +52,5 @@ endif
 ifeq ($(findstring $(ARCH),$(i686_ARCHS)),$(ARCH))
 CMAKE_ARGS += -DAOM_TARGET_CPU=x86
 endif
+
+ENV += CXXFLAGS=-D_GLIBCXX_USE_C99_MATH
```

Comment:
It does not build under `armv7`.  Also I propose using standard CMAKE calls.
I propose the following instead, added :
```
include ../../mk/spksrc.cross-cmake.mk

# libaom specific cmake options
CMAKE_ARGS += -DCONFIG_AV1_ENCODER=1

ifeq ($(call version_ge, ${TCVERSION}, 7.0),1)
CMAKE_ARGS += -DCMAKE_CXX_FLAGS=-D_GLIBCXX_USE_C99_MATH
ifeq ($(findstring $(ARCH),$(ARMv7_ARCHS) $(ARMv7L_ARCHS)),$(ARCH))
CMAKE_ARGS += -DENABLE_NEON=OFF
endif
endif
```